src/main/java/racingcar/RacingCarMain.java

```diff
@@ -6,17 +6,24 @@
 public class RacingCarMain {
 
     public static void main(String[] args) {
-        int carNum = InputView.getCarNumber();
+        String[] names = InputView.getCarNames();
         int roundNum = InputView.getRoundNumber();
 
         List<Car> cars = new ArrayList<>();
-        for (int i = 0; i < carNum; i++) {
-            cars.add(new Car());
+        for (int i = 0; i < names.length; i++) {
+            cars.add(new Car(names[i]));
         }
 
         ResultView resultView = new ResultView();
-        Racing racing = new Racing(cars, roundNum, resultView);
-        racing.start();
+        Racing racing = new Racing(cars);
+
+        for (int i = 0; i < roundNum; i++) {
+            resultView.roundStart(i + 1);
```

Comment:
정적 메서드를 호출하는 거라면, 지금처럼 인스턴스를 생성할 필요가 없지 않을까요?
```suggestion
            ResultView.roundStart(i + 1);
```